README.md

```diff
@@ -159,6 +159,7 @@ of this pipeline:
 Those who have provided conceptual guidance, suggestions, bug reports etc.
 
 * Arielle Munters
+* [Charles Plessey](https://github.com/charles-plessy)
```

Comment:
Sorry there is a typo in my name... It is _Plessy_.

Reply:
Fixed it!